src/decisionengine/framework/taskmanager/TaskManager.py

```diff
@@ -108,11 +110,19 @@ def __init__(self, channel_dict, channel_name):
 
         delogger.debug("Creating channel source")
         self.sources = _make_workers_for(channel_dict["sources"], Source, channel_name)
-        delogger.debug("Creating channel logicengine")
-        self.le_s = _make_workers_for(channel_dict["logicengines"], LogicEngine, channel_name)
+
         delogger.debug("Creating channel publisher")
         self.publishers = _make_workers_for(channel_dict["publishers"], Publisher, channel_name)
 
+        delogger.debug("Creating channel logicengine")
+        configured_le_s = channel_dict.get("logicengines")
+        if configured_le_s is None:
+            configured_le_s = passthrough_configuration(channel_dict["publishers"].keys())
```

Comment:
Should we add a `debug` level log saying "we loaded the passthrough"?

Reply:
Sounds good--will make the change soon.  On a different note, do you know why the pre-commit linters check is failing?  I've rebased this PR on top of all of the `master` commits.  Looks like we're being bit by whitespaces again...?

Reply:
yep, white space is hitting us.

I've got it fixed up in #444